client/app/components/Input/__tests__/InputPassword.spec.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { mount } from 'enzyme';
+import { InputMocks } from '../../../mocks/InputMocks';
+import { shallow } from 'enzyme';
+import React from 'react';
+import { InputPassword } from '../InputPassword';
+
+const text = 'some-password-text';
+const component = InputMocks.createInput(InputMocks.inputPasswordProps);
+const input = `input#${InputMocks.inputPasswordProps.id}`;
+
+  describe('on mouse click', () => {
```

Comment:
Let's fix the indentation of this file. We should name this describe block `InputPassword` instead to match the convention of our other test files.